app/Models/Post.php

```diff
@@ -34,6 +34,7 @@
  * @property bool|int $is_private
  * @property int $deleted_by
  * @property bool|int $is_pinned
+ * @property bool $is_template
```

Comment:
I think it needs to also be cast as int for phpstan, but could be wrong (that's why is_pinned and is_private are both bool|int